package.json

```diff
@@ -59,16 +59,16 @@
         "easymde": "^2.18.0",
         "esbuild": "^0.25.0",
         "esbuild-postcss": "^0.0.4",
-        "filepond": "^4.31.1",
-        "filepond-plugin-file-validate-size": "^2.2.4",
-        "filepond-plugin-file-validate-type": "^1.2.6",
+        "filepond": "^4.32.7",
```

Comment:
Updated to the latest version as there were some bug fixes across all the filepond packages